spec/ui_spec.rb

```diff
@@ -85,6 +85,43 @@ def app
     expect(response.body).to eq 'Hi!'
   end
 
+  it 'resets the number of times a request has been made to an endpoint' do
+    get '/endpoint'
+    get '/endpoint'
+    get '/endpoint'
+
+    visit '/'
+
+    expect(page).to have_css 'tr', text: '3'
+
+    within 'tr', text: '/endpoint' do
+      click_on 'Reset'
+    end
+
+    visit '/'
+
+    expect(page).to_not have_css 'tr', text: 3
+  end
+
+  it 'resets the body of custom matchers' do
+    visit '/'
+
+    within 'tr', text: '/key1' do
+      click_on '/matcher'
+    end
+    fill_in 'status', :with => '204'
+    fill_in 'body', :with => 'i am a llama'
+    click_on 'Save'
+
+    within 'tr', text: '/key1' do
+      click_on 'Reset'
+      click_on '/matcher'
+    end
+
+    expect(page).to have_selector 'input[value="202"]'
```

Comment:
You can use `expect(page).to have_field 'Status', with: 202` for things like this.